includes/admin/class-wc-stripe-admin-notices.php

```diff
@@ -175,9 +175,9 @@ public function stripe_check_environment() {
 
 			if ( empty( $show_ssl_notice ) ) {
 				// Show message if enabled and FORCE SSL is disabled and WordpressHTTPS plugin is not detected.
-				if ( ( function_exists( 'wc_site_is_https' ) && ! wc_site_is_https() ) && ( 'no' === get_option( 'woocommerce_force_ssl_checkout' ) && ! class_exists( 'WordPressHTTPS' ) ) ) {
+				if ( function_exists( 'wc_site_is_https' ) && ! wc_site_is_https() ) {
 					/* translators: 1) link 2) link */
-					$this->add_admin_notice( 'ssl', 'notice notice-warning', sprintf( __( 'Stripe is enabled, but the <a href="%1$s">force SSL option</a> is disabled; your checkout may not be secure! Please enable SSL and ensure your server has a valid <a href="%2$s" target="_blank">SSL certificate</a> - Stripe will only work in test mode.', 'woocommerce-gateway-stripe' ), admin_url( 'admin.php?page=wc-settings&tab=checkout' ), 'https://en.wikipedia.org/wiki/Transport_Layer_Security' ), true );
+					$this->add_admin_notice( 'ssl', 'notice notice-warning', sprintf( __( 'Stripe is enabled, but the <a href="%1$s">SSL</a> is not detected. Your checkout may not be secure! Please enable SSL and ensure your server has a valid <a href="%2$s" target="_blank">SSL certificate</a> - Stripe will only work in test mode.', 'woocommerce-gateway-stripe' ), admin_url( 'admin.php?page=wc-settings&tab=checkout' ), 'https://en.wikipedia.org/wiki/Transport_Layer_Security' ), true );
```

Comment:
`Stripe is enabled, but the <a href="%1$s">SSL</a> is not detected` that wording is a bit awkward maybe just remove `the` so its `Stripe is enabled, but <a href="%1$s">SSL</a> is not detected`

Reply:
Revised.